packages/in_app_purchase/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: in_app_purchase
 description: A Flutter plugin for in-app purchases.
 author:  Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase
-version: 0.0.2
+version: 0.0.1
```

Comment:
It turns out this was done on purpose, I'm ok with both options (keeping it 0.0.1 or 0.0.2), but make sure @gspencergoog is ok with that as well.

Reply:
Oh, interesting. What's the reason for having it set to `0.0.2`?

Reply:
It's to track dependency updates, I'll keep it at 2.